app/controllers/ips_controller.rb

```diff
@@ -37,34 +38,21 @@ def destroy
 private
 
   def available_locations
-    current_organisation.locations.order('address ASC').map do |loc|
-      ["#{loc.address}, #{loc.postcode}", loc.id]
-    end
+    current_organisation.locations.order(:address)
   end
 
   def ip_params
     params.require(:ip).permit(:address, :location_id, location_attributes: %i[address postcode])
   end
 
   def create_params
-    return params_with_new_location if user_creates_new_location?
-    params_with_existing_location
+    ip_params.except(:location_attributes)
   end
 
   def user_creates_new_location?
```

Comment:
This method doesn't seem to be used elsewhere. It could be removed.

Reply:
Good spot! I'll get rid of it.